x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_suggestions.ts

```diff
@@ -20,6 +20,7 @@ export function getSuggestions({
   // We only render metric charts for single-row queries. We require a single, numeric column.
   if (
     table.isMultiRow ||
+    table.columns.length === 0 ||
```

Comment:
Line 23 and 24 could be united into `table.columns.length >= 0 ||` or just `table.columns.length ||`

Reply:
You're right that they can be combined, but the condition is actually `table.columns.length !== 1`